packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+      "IMPORT_USER_SCHEMA",
+      "PUSH_NEW_USERS",
+      "PUSH_PASSWORD_UPDATES",
+      "PUSH_PROFILE_UPDATES",
+      "PUSH_USER_DEACTIVATION",
+      "REACTIVATE_USERS"
+    ],
+    "_links": {
+      "logo": [
+        {
+          "name": "medium",
+          "href": "https://${yourOktaDomain}/img/logos/salesforce_logo.png",
+          "type": "image/png"
+        }
+      ],
+      "self": {
+          "href": "https://${yourOktaDomain}/api/v1/catalog/apps/salesforce"
+      }
+    }
+  },
+  {
+    "name": "boxnet",
+    "displayName": "Box",
+    "description": "Cloud storage.",
+    "status": "ACTIVE",
+    "lastUpdated": "2014-06-03T16:17:13.000Z",
+    "category": "CM",
+    "verificationStatus": "OKTA_VERIFIED",
+    "website": "http://www.box.net",
+    "signOnModes": [
+      "SAML_2_0"
+    ],
+    "features": [
+      "GROUP_PUSH",
+      "IMPORT_NEW_USERS",
+      "IMPORT_PROFILE_UPDATES",
+      "PUSH_NEW_USERS",
+      "PUSH_PROFILE_UPDATES",
+      "PUSH_USER_DEACTIVATION",
+      "REACTIVATE_USERS"
+    ],
+    "_links": {
+      "logo": [
+        {
+          "name": "medium",
+          "href": "https://${yourOktaDomain}/img/logos/box.png",
+          "type": "image/png"
+        }
+      ],
+      "self": {
+          "href": "https://${yourOktaDomain}/api/v1/catalog/apps/boxnet"
+      }
+    }
+  },
+  {
+    "name": "Facebook for Detroit Office",
+    "status": "ACTIVE",
+    "id": "0oapsqQ5dv19pqyEo0g3",
+    "_links": {
+      "self": {
+          "href": "https://${yourOktaDomain}/api/v1/apps/0oapsqQ5dv19pqyEo0g3"
+      }
+    }
+  },
+  {
+    "name": "Facebook (Toronto)",
+    "status": "ACTIVE",
+    "id": "0obdfgrQ5dv29pqyQo0f5",
+    "_links": {
+       "self": {
+           "href": "https://${yourOktaDomain}/api/v1/apps/0obdfgrQ5dv29pqyQo0f5"
+       }
+    }
+  }
+]
+```
+
+##### List App targets for App Administrator Role given to a Group
+
+
+<ApiOperation method="get" url="/api/v1/groups/${groupId}/roles/${roleId}/targets/catalog/apps" />
+
+Lists all App targets for an `APP_ADMIN` Role assigned to a Group
+
+###### Request Parameters
+
+
+| Parameter   | Description                                                    | Param Type   | DataType   | Required |
+| :---------- | :------------------------------------------------------------- | :----------- | :--------- | :------- |
+| after       | Specifies the pagination cursor for the next page of targets   | Query        | String     | FALSE    |
+| groupId     | `id` of a Group                                                | URL          | String     | TRUE     |
+| limit       | Specifies the number of results for a page (default is 20)     | Query        | Number     | FALSE    |
+| roleId      | `id` of a Role                                                 | URL          | String     | TRUE     |
+
+Treat the page cursor as an opaque value: obtain it through the next link relation. See [Pagination](/docs/reference/api-overview/#pagination).
```

Comment:
Treat the page cursor as an opaque value: obtain it through the next link relation. > Treat the page cursor as an opaque value. You can obtain it through the next link relation.